browser-sign-in/__tests__/App-test.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @format
+ */
+
+//import 'react-native';
+//import React from 'react';
+//import App from '../App';
+
+//import { shallow, render } from 'enzyme';
+
+// Note: test renderer must be required after react-native.
+//import renderer from 'react-test-renderer';
+// import { 
+// 	createConfig, 
+//   signIn, 
+//   signOut, 
+//   getAccessToken, 
+//   isAuthenticated,
+//   getUser,
+//   getUserFromIdToken
+// } from '@okta/okta-react-native';
+
+// jest.mock('@okta/okta-react-native', () => {
+// 	// const EventEmitter = {
+// 	// 	addEventListener: jest.fn((event, cb) => {
+// 	// 		cb();
+// 	// 	})
+// 	// };
+//   return { 
+// 		createConfig: jest.fn(),
+// 		signIn: jest.fn(),
+// 		signOut: jest.fn(),
+// 		getAccessToken: jest.fn(),
+// 		isAuthenticated: jest.fn(),
+// 		getUser: jest.fn(),
+// 		getUserFromIdToken: jest.fn(),
+// 		//EventEmitter
+//   };
+// });
+
+// global.fetch = jest
```

Comment:
Are you using these tests?  
I'd suggest either removing them, or having the tests that are temporarily disabled (if any) noted with xdescribe() and xit().
If you're leaving commented out code for some reason, best to put a comment as to why.

Reply:
The tests will be updated in another PR, i was just keeping them for reference lol... wrote some tests but decided to put another PR up for that